docs/reference/inference/put-inference.asciidoc

```diff
@@ -266,8 +303,20 @@ maximum token length. Defaults to `END`. Valid values are:
 
 `user`:::
 (optional, string)
+<<<<<<< HEAD
```

Comment:
Some conflict indicators remained in the commit. Please remove them.

Reply:
oof - good catch. Thanks!